pkg/plugin/submariner/install.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright Kurator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package submariner
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"path"
+
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+
+	"kurator.dev/kurator/pkg/util"
+)
+
+func (p *SubmarinerPlugin) runInstall() error {
+	// 1.generate kubeconfig for each cluster
+	memClusters, err := p.generateKubeConfiguration()
+	if err != nil {
+		return err
+	}
+	if len(memClusters) < 2 {
+		return fmt.Errorf("memcluster number %d < 2", len(memClusters))
+	}
+	// 2. install broker in the first memcluster
+	err = p.installBroker(memClusters[0])
+	if err != nil {
+		return fmt.Errorf("install broker failed: %v", err)
+	}
+	// 3. connect all clusters to the Broker
+	err = p.connectBroker(memClusters)
```

Comment:
the cluster which broker installed in also need connect to broker?

Reply:
Seems need to